### ISSUE TITLE
More than one activity per task

### ISSUE THREAD

**Issue by chrisw14** [imixs/imixs-workflow#368]
Hi Ralph,
does the imixs workflow engine support more than one activity per task?
I mean that there are some editors at one task (namOwner) which have to do some activities at the same document, task and time. If all activities have been done the workitem is forwarded to the next task.
Is it possible to define these activities which belongs to one task in the bpmn model?
I tried it with the VersionPlugin but I don't want to have two documents...

Thank you!
Chris

**Comment by rsoika** [comment]
Yes we have something like this called [ApproverPlugin](https://www.imixs.org/marty/plugins/approverplugin.html). It is poorly documented so you better look into sources ;-)

The idea is to provide a field with owners. Each time one of this owners is processing the task, this owner is put into the [namfileld]approvedby item. So you can monitor when all approvers are finished.
